src/argilla/server/models/database.py

```diff
@@ -167,6 +181,27 @@ def __repr__(self):
         )
 
 
+class MetadataProperty(DatabaseModel):
+    __tablename__ = "metadata_properties"
+
+    name: Mapped[str] = mapped_column(Text, index=True)
+    type: Mapped[MetadataPropertyType] = mapped_column(Text)
```

Comment:
```suggestion
    type: Mapped[MetadataPropertyType] = mapped_column(MetadataPropertyTypeEnum)
```

Reply:
I prefer to keep it as Text and check the enum at the application level. We had some problems updating enums in the past and this could be an entry point for future problems (with other db than SQLite)